<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDialog</class>
 <widget class="QDialog" name="SettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>452</width>
    <height>506</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="Decoding">
      <attribute name="title">
       <string>Decoding</string>
      </attribute>
      <widget class="QGroupBox" name="groupBox">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>401</width>
         <height>71</height>
        </rect>
       </property>
       <property name="title">
        <string>Do not display SU types</string>
       </property>
       <widget class="QLineEdit" name="lineEditdonotdisplaysus">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>381</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>26 0A</string>
        </property>
       </widget>
      </widget>
      <widget class="QGroupBox" name="groupBox_4">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>90</y>
         <width>401</width>
         <height>101</height>
        </rect>
       </property>
       <property name="title">
        <string>ACARS window</string>
       </property>
       <widget class="QCheckBox" name="checkBoxdropnontextmsgs">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>68</y>
          <width>201</width>
          <height>19</height>
         </rect>
        </property>
        <property name="text">
         <string>Drop non text messages</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_8">
        <property name="geometry">
         <rect>
          <x>25</x>
          <y>28</y>
          <width>48</width>
          <height>25</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>25</height>
         </size>
        </property>
        <property name="text">
         <string>Format</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
       <widget class="QComboBox" name="comboBoxDisplayformat">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>30</y>
          <width>38</width>
          <height>21</height>
         </rect>
        </property>
        <property name="maximumSize">
         <size>
          <width>40</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <item>
         <property name="text">
          <string>1</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>2</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>3</string>
         </property>
        </item>
       </widget>
      </widget>
      <widget class="QGroupBox" name="groupBox_5">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>280</y>
         <width>401</width>
         <height>111</height>
        </rect>
       </property>
       <property name="title">
        <string>Soundcard</string>
       </property>
       <widget class="QComboBox" name="comboBoxsoundcard">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>381</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QCheckBox" name="checkBoxlogwidebandwidthenable">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>70</y>
          <width>321</width>
          <height>19</height>
         </rect>
        </property>
        <property name="text">
         <string>Enable widebandwidth (CPU intensive)</string>
        </property>
       </widget>
      </widget>
      <widget class="QGroupBox" name="groupBox_6">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>200</y>
         <width>401</width>
         <height>71</height>
        </rect>
       </property>
       <property name="title">
        <string>Logging</string>
       </property>
       <widget class="QLineEdit" name="lineEditlogdir">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>30</y>
          <width>201</width>
          <height>21</height>
         </rect>
        </property>
       </widget>
       <widget class="QCheckBox" name="checkBoxlogenable">
        <property name="geometry">
         <rect>
          <x>320</x>
          <y>30</y>
          <width>71</width>
          <height>19</height>
         </rect>
        </property>
        <property name="text">
         <string>Enable</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_9">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>26</y>
          <width>81</width>
          <height>25</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>25</height>
         </size>
        </property>
        <property name="text">
         <string>Directory</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QWidget" name="logwindowsettings">
      <attribute name="title">
       <string>Log Window</string>
      </attribute>
      <widget class="QLineEdit" name="lineEditplanelookup">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>28</y>
         <width>291</width>
         <height>21</height>
        </rect>
       </property>
       <property name="toolTip">
        <string>URL for plane lookup. {AES} is the plane ICAO number in hex.</string>
       </property>
       <property name="text">
        <string>http://junzisun.com/aif/?q={AES}#</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>101</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Plane lookup</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEditplanesfolder">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>60</y>
         <width>291</width>
         <height>21</height>
        </rect>
       </property>
       <property name="toolTip">
        <string>Planes images are stored as {AES}.png or {AES}.jpg and database as aircrafts_dump.csv in this folder.</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>101</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Plane folder</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="Transmission">
      <attribute name="title">
       <string>Transmission</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="7">
         <item>
          <widget class="QGroupBox" name="groupBox_3">
           <property name="title">
            <string>RF</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <layout class="QFormLayout" name="formLayout">
              <item row="0" column="0">
               <widget class="QLabel" name="label_3">
                <property name="text">
                 <string>Freq</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QSpinBox" name="spinBoxTXFreq">
                <property name="buttonSymbols">
                 <enum>QAbstractSpinBox::NoButtons</enum>
                </property>
                <property name="showGroupSeparator" stdset="0">
                 <bool>true</bool>
                </property>
                <property name="suffix">
                 <string>Hz</string>
                </property>
                <property name="minimum">
                 <number>100</number>
                </property>
                <property name="maximum">
                 <number>100000000</number>
                </property>
                <property name="value">
                 <number>1000</number>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Ambles</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <layout class="QFormLayout" name="formLayout_2">
            <item row="0" column="0">
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Pre</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>Post</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0" colspan="2">
             <layout class="QHBoxLayout" name="horizontalLayout">
              <item>
               <widget class="QLabel" name="label_4">
                <property name="text">
                 <string>Minimum preamble</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QDoubleSpinBox" name="doubleSpinboxminpreamble">
                <property name="buttonSymbols">
                 <enum>QAbstractSpinBox::NoButtons</enum>
                </property>
                <property name="suffix">
                 <string>s</string>
                </property>
                <property name="maximum">
                 <double>10.000000000000000</double>
                </property>
                <property name="value">
                 <double>1.500000000000000</double>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>88</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
            <item row="1" column="1">
             <widget class="QLineEdit" name="linePostamble"/>
            </item>
            <item row="0" column="1">
             <widget class="QLineEdit" name="linePreamble"/>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>9</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
